app/models/child.rb

```diff
@@ -122,6 +122,10 @@ def pending_envelope?
     childcare_envelopes.pluck(:status).any? { |s| ChildcareEnvelope::IN_PROCESS_ENVELOPE_STATUSES.include?(s) }
   end
 
+  def send_docusign_envelope
+    Docusign::SendChildcareEnvelopeJob.perform_later(self)
```

Comment:
I'd recommend considering removing references to job classes from the model layer, the reason is because it creates a circular dependency among the classes (the model depends on job, and the job depends on the model).